website/docs/blog.md

```diff
@@ -63,6 +64,7 @@ The only required field is `title`; however, we provide options to add author in
 - `draft` - A boolean flag to indicate that the blog post is work in process and therefore should not be published yet. However, draft blog posts will be displayed during development.
 - `description`: The description of your post, which will become the `<meta name="description" content="..."/>` and `<meta property="og:description" content="..."/>` in `<head>`, used by search engines. If this field is not present, it will default to the first line of the contents.
 - `image`: Cover or thumbnail image that will be used when displaying the link to your post.
+- `hide_table_of_contents`: Whether to hide the table of contents to the right. By default it is `false`.
```

Comment:
I think we don't need this option for now. 
We don't support hiding the docs TOC neither and nobody complained so far, and it's a part of v1 on which we want feature parity anyway.

If we were to provide options to hide the TOC, we'd rather add this as themeConfig, and also make the docs TOC configurable, because adding this frontmatter to every blog post is likely not very convenient. Users will likely want the same setting on ALL the blog posts.

(Also naming using _ is not consistent with snakeCase we already use in other options)

Reply:
Ah I see. Is the website out of sync then? I saw [here](https://v2.docusaurus.io/docs/markdown-features#markdown-headers) that was a `hide_table_of_contents` field for docs and that the `DocItem` component accounted for it. But if that's the case, I think it makes sense to remove it too!

Reply:
Ah this is my mistake, if docs has `hide_table_of_contents` then we should add the same to blog you are right ;)